packages/editor/src/plugins/exercise/static.tsx

```diff
@@ -1,26 +1,50 @@
 import { StaticRenderer } from '@editor/static-renderer/static-renderer'
 import { EditorExerciseDocument } from '@editor/types/editor-plugins'
 import { isRowsDocument } from '@editor/types/plugin-type-guards'
+import { faCircleCheck } from '@fortawesome/free-regular-svg-icons'
+import { useState } from 'react'
 
 import { isEmptyTextDocument } from '../text/utils/static-is-empty'
+import { FaIcon } from '@/components/fa-icon'
+import { useInstanceData } from '@/contexts/instance-context'
 
 export function ExerciseStaticRenderer({ state }: EditorExerciseDocument) {
-  const { content, interactive, solution } = state
+  const { content, interactive, solution, hideInteractive } = state
+  const [revealInteractive, setRevealInteractive] = useState(
```

Comment:
Renaming suggestion: `interactiveHidden`
Together with the other renaming it could read like `const [interactiveHidden, setInteractiveHidden] = useState(hideInteractiveInitially)`